packages/esm-patient-immunizations-app/src/immunizations/immunization-history-card.scss

```diff
@@ -0,0 +1,70 @@
+@use '@carbon/type';
+@use '@openmrs/esm-styleguide/src/vars' as *;
+
+.widgetCard {
+  border: 1px solid $ui-03;
+}
+
+.content {
+  background-color: $ui-background;
+}
+
+.doseCell {
+  min-width: 140px;
+  padding: 0.5rem;
+  display: inline-block;
+  text-align: center;
+}
+
+.doseLabel {
+  margin-bottom: 0.5rem;
+}
+
+.doseDate {
+  font-size: 0.875rem;
+  color: $text-02;
```

Comment:
You should be able to use `type.$label-02` (which adjusts other factors we should adjust here.